yamscripts.yml

```diff
@@ -21,11 +21,10 @@ scripts:
     webpack: webpack --config webpack.system-addon.config.js --display-optimization-bailout
     css: node-sass --source-map true --source-map-contents system-addon/content-src/styles -o system-addon/css
     html: rimraf system-addon/prerendered && webpack --config webpack.prerender.config.js && node ./bin/render-activity-stream-html.js
-    # Copy over all of the system-addon directory
-    copy: cpx "system-addon/**/{,.}*" $npm_package_config_mc_dir/browser/extensions/activity-stream
-    version: node ./bin/update-version.js $npm_package_config_mc_dir/browser/extensions/activity-stream
+    # Copy over the entire repo
+    copy: cpx "./**/{,.}*" $npm_package_config_mc_dir/browser/extensions/activity-stream
```

Comment:
This seems wrong to copy over node_modules (even if ignored/not-checked-in-to-m-c) and other activity-stream "built" files/directories that are not used as part of the export. I.e., the mirror should be like a clean checkout of the activity-stream repository with maybe the desired built files.

Reply:
In particular, it's trying to copy over 200MB+ of node_modules files to ignore (or worse(?) -- overwrite)